lib/segment/src/index/field_index/bool_index/mmap_bool_index.rs

```diff
@@ -230,11 +309,15 @@ impl PayloadFieldIndex for MmapBoolIndex {
         let Self {
             base_dir: _,
             indexed_count,
-            trues_slice: _,
-            falses_slice: _,
+            trues_count,
+            falses_count,
+            trues_slice,
+            falses_slice,
         } = self;
 
         *indexed_count = calculated_indexed_count as usize;
```

Comment:
Can't we replace this one with `trues_count + falses_count` and remove the calculation above?

Reply:
no, because `indexed_count` is the count of the union of `trues_slice` and `falses_slice`